test/manual/amp-analytics/analytics-linker.html

```diff
@@ -0,0 +1,70 @@
+<!doctype html>
```

Comment:
I think more examples are good, but for future reference there are a couple files, `examples/linkers.html`, `/test/manual/amp-linker-simple.amp.html`

Reply:
Thanks for the reference.